be/src/exec/jdbc_scanner.cpp

```diff
@@ -356,6 +356,14 @@ StatusOr<LogicalType> JDBCScanner::_precheck_data_type(const std::string& java_c
                                 slot_desc->col_name(), java_class));
         }
         return TYPE_VARCHAR;
+    } else if (java_class == "byte[]" || java_class == "oracle.jdbc.OracleBlob" || java_class == "[B") {
```

Comment:
why do `byte[]` and `[B` coexist. Under what conditions is `byte[]`, under what conditions is `byte[]`

Reply:
Yes, they are different in name, but the same object. I find out that resultset get from oracle and sqlserver are different in name, maybe it's related to the driver.